seatunnel-connectors-v2/connector-hudi/src/main/java/org/apache/seatunnel/connectors/seatunnel/hudi/sink/convert/HudiRecordConverter.java

```diff
@@ -62,7 +63,9 @@ public HoodieRecord<HoodieAvroPayload> convertRow(
                     seaTunnelRowType.getFieldNames()[i],
                     createConverter(seaTunnelRowType.getFieldType(i))
                             .convert(
-                                    convertToSchema(seaTunnelRowType.getFieldType(i)),
+                                    convertToSchema(
+                                            seaTunnelRowType.getFieldType(i),
+                                            ROW_NAME + "_" + seaTunnelRowType.getFieldNames()[i]),
```

Comment:
Is this a standrad way of set field name? `org.apache.seatunnel.avro.generated.record_fieldname`?

Reply:
> Is this a standrad way of set field name? `org.apache.seatunnel.avro.generated.record_fieldname`?

I have modified the schema generation logic according to hudi official. PTAL. @liugddx @Hisoka-X .